### DISCUSSION TITLE
Is there a way to list the available alias of a catalog

### DISCUSSION THREAD

**Discussion by grumpyf0x48** [jbangdev/jbang#740] [Ideas] [answered]
It would be great to see in the terminal the aliases and their description defined in a catalog:

$ jbang catalog list-content

**Comment by grumpyf0x48** [comment]
$ jbang alias list

but it does not say to which catalog each alias belongs to.

**Reply by maxandersen** [reply]
`jbang alias list --show-origin`

then the catalog will be printed above the aliases it defines.

If you know the catalog you can do:

`jbang catalog list grumpyf0x48`

and it list the aliases as defined in that specific catalog.

you can also add `--verbose` if you want to see how the resolving happens; but that is mainly used for debugging if some folder in hierarchy are somehow overshadowing some other alias.

**Reply by grumpyf0x48** [reply]
Thanks a lot.

**Comment by quintesse** [comment, marked as answer]
`jbang alias list --show-origin`

then the catalog will be printed above the aliases it defines.

If you know the catalog you can do:

`jbang catalog list grumpyf0x48`

and it list the aliases as defined in that specific catalog.

you can also add `--verbose` if you want to see how the resolving happens; but that is mainly used for debugging if some folder in hierarchy are somehow overshadowing some other alias.

(copied from @maxandersen 's comment)
